qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -1060,17 +1147,44 @@ def load_parameter_values(self, file_name: str = "parameter_values.csv"):
             reader = csv.DictReader(fp, delimiter=",", quotechar='"')
 
             for row in reader:
-                param_val = ParameterValue(
-                    row["value"], row["date_time"], row["valid"], row["exp_id"], row["group"]
-                )
+                self.add_parameter_value_from_conf(**row)
 
-                if row["schedule"] == "":
-                    schedule_name = None
-                else:
-                    schedule_name = row["schedule"]
+    def add_parameter_value_from_conf(
```

Comment:
probably protected method? or is this intended to be exposed to users?

Reply:
Protected is fine. See here: 562031d